packages/scenario/test/bind-types.test.ts

```diff
@@ -616,6 +644,62 @@ describe('Data (<bind type>) type support', () => {
 				}
 			);
 		});
+
+		describe('type="date"', () => {
```

Comment:
This is a good place to start reviewing and see if the encode/decode is doing what we expect.